components/logins/src/sync/engine.rs

```diff
@@ -364,6 +364,17 @@ impl LoginsSyncEngine {
         Ok(())
     }
 
+
+    pub fn delete_local_sync_data(&self) -> Result<()> {
```

Comment:
I kept this separate function because I'm not cloning the mirror, resetting the sync status of `LoginL` records, which occurs in the `do_reset` function. However, I have no problems removing this function and calling `do_reset(EngineSyncAssociation::Disconnected)` if that makes more sense.